src/uu/split/src/split.rs

```diff
@@ -20,80 +24,122 @@ use std::path::Path;
 
 static NAME: &str = "split";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
-
-pub fn uumain(args: impl uucore::Args) -> i32 {
-    let args = args.collect_str();
-
-    let mut opts = getopts::Options::new();
-
-    opts.optopt(
-        "a",
-        "suffix-length",
-        "use suffixes of length N (default 2)",
-        "N",
-    );
-    opts.optopt("b", "bytes", "put SIZE bytes per output file", "SIZE");
-    opts.optopt(
-        "C",
-        "line-bytes",
-        "put at most SIZE bytes of lines per output file",
-        "SIZE",
-    );
-    opts.optflag(
-        "d",
-        "numeric-suffixes",
-        "use numeric suffixes instead of alphabetic",
-    );
-    opts.optopt(
-        "",
-        "additional-suffix",
-        "additional suffix to append to output file names",
-        "SUFFIX",
-    );
-    opts.optopt(
-        "",
-        "filter",
-        "write to shell COMMAND file name is $FILE (Currently not implemented for Windows)",
-        "COMMAND",
-    );
-    opts.optopt("l", "lines", "put NUMBER lines per output file", "NUMBER");
-    opts.optflag(
-        "",
-        "verbose",
-        "print a diagnostic just before each output file is opened",
-    );
-    opts.optflag("h", "help", "display help and exit");
-    opts.optflag("V", "version", "output version information and exit");
-
-    let matches = match opts.parse(&args[1..]) {
-        Ok(m) => m,
-        Err(f) => crash!(1, "{}", f),
-    };
-
-    if matches.opt_present("h") {
-        let msg = format!(
-            "{0} {1}
-
-Usage:
+// TODO: pack OPTION_… into local module?
```

Comment:
Can you do that in alphabetic order please ? thanks!

Reply:
fo sho